networkx/algorithms/threshold.py

```diff
@@ -32,22 +29,30 @@ def is_threshold_graph(G):
     >>> is_threshold_graph(G)
     False
 
-    References
-    ----------
-    .. [1] Threshold graphs: https://en.wikipedia.org/wiki/Threshold_graph
+    Returns
+    -------
+    True if `G` is a threshold graph, `False` otherwise.
     """
     return is_threshold_sequence(list(d for n, d in G.degree()))
 
 
 def is_threshold_sequence(degree_sequence):
-    """
-    Returns True if the sequence is a threshold degree seqeunce.
+    """Returns True if the sequence is a threshold degree seqeunce.
 
     Uses the property that a threshold graph must be constructed by
     adding either dominating or isolated nodes. Thus, it can be
     deconstructed iteratively by removing a node of degree zero or a
     node that connects to the remaining nodes.  If this deconstruction
     failes then the sequence is not a threshold sequence.
+
+    Parameters
+    ----------
+    degree_sequence
+        The degree sequence
+
+    Returns
+    -------
+    False if not a threshold degree sequence, True otherwise
```

Comment:
This should conform to the [numpydoc standard](https://numpydoc.readthedocs.io/en/latest/format.html#returns) e.g. something like:

```suggestion
    Returns
    -------
    bool
        Whether `degree_sequence` is a threshold sequence.
```